examples/draw-features.html

```diff
@@ -6,17 +6,18 @@
   Example of using the Draw interaction. Select a geometry type from the
   dropdown above to start drawing. To finish drawing, click the last
   point. To activate freehand drawing for lines, polygons, and circles, hold
-  the `Shift` key.
+  the `Shift` key. To remove the last point of a line or polygon, press "Undo".
 tags: "draw, edit, freehand, vector"
 ---
 <div id="map" class="map"></div>
 <form class="form-inline">
-  <label for="type">Geometry type &nbsp;</label>
-  <select id="type">
+  <label for="type">Geometry type: &nbsp;</label>
+  <select class="form-control mr-2 mb-2 mt-2" id="type">
     <option value="Point">Point</option>
     <option value="LineString">LineString</option>
     <option value="Polygon">Polygon</option>
     <option value="Circle">Circle</option>
     <option value="None">None</option>
   </select>
+  <input class="form-control mr-2 mb-2 mt-2" type="button" value="Undo" id="undo"></input>
```

Comment:
```suggestion
  <input class="form-control mr-2 mb-2 mt-2" type="button" value="Undo" id="undo">
```

Input is a void element.